Description: Use a different file for testing instead of unavailable Cargo.lock
 .
 The transfer-encoding check in async_impl_file_part() was removed in upstream
 commit 449f251bd2311a8ba73d550c6da3d3ff39d04ce8.
Last-Update: 2024-11-02
--- a/tests/multipart.rs
+++ b/tests/multipart.rs
@@ -127,6 +127,6 @@
     let form = reqwest::blocking::multipart::Form::new()
-        .file("foo", "Cargo.lock")
+        .file("foo", "LICENSE-APACHE")
         .unwrap();
 
-    let fcontents = std::fs::read_to_string("Cargo.lock").unwrap();
+    let fcontents = std::fs::read_to_string("LICENSE-APACHE").unwrap();
 
@@ -135,3 +135,3 @@
          --{0}\r\n\
-         Content-Disposition: form-data; name=\"foo\"; filename=\"Cargo.lock\"\r\n\
+         Content-Disposition: form-data; name=\"foo\"; filename=\"LICENSE-APACHE\"\r\n\
          Content-Type: application/octet-stream\r\n\r\n\
@@ -184,3 +184,3 @@
     let form = reqwest::multipart::Form::new()
-        .file("foo", "Cargo.lock")
+        .file("foo", "LICENSE-APACHE")
         .await
@@ -188,3 +188,3 @@
 
-    let fcontents = std::fs::read_to_string("Cargo.lock").unwrap();
+    let fcontents = std::fs::read_to_string("LICENSE-APACHE").unwrap();
 
@@ -193,3 +193,3 @@
          --{0}\r\n\
-         Content-Disposition: form-data; name=\"foo\"; filename=\"Cargo.lock\"\r\n\
+         Content-Disposition: form-data; name=\"foo\"; filename=\"LICENSE-APACHE\"\r\n\
          Content-Type: application/octet-stream\r\n\r\n\
@@ -210,3 +210,2 @@
             assert_eq!(req.headers()["content-type"], ct);
-            assert_eq!(req.headers()["transfer-encoding"], "chunked");
 
